packages/api/schema/stryker-core.json

```diff
@@ -765,7 +765,7 @@
       "title": "UnaryOperator",
       "type": "array",
       "default": [],
-      "minItems": 1,
+      "minItems": 0,
```

Comment:
Just for consistency sake, the other operators have the minItems set to 1. Maybe it would be best to wait for the form responses until we decide if we will accept a syntax like "UnaryOperator: []" ?